apps/web-giddh/src/app/actions/comprehensive-constants.ts

```
@@ -0,0 +1,1031 @@
+};
+
+// SubscriptionsActions constants
+export const SubscriptionsActions = {
+    SubscribedCompaniesResponse: 'SubscribedCompaniesResponse',
+    SubscribedUserTransactionsResponse: 'SubscribedUserTransactionsResponse',
+    SubscribedCompanyTransactionsResponse: 'SubscribedCompanyTransactionsResponse',
+    SubscribedCompaniesListResponse: 'SubscribedCompaniesListResponse',
+};
+
+// AgingReportActions class
+export class AgingReportActions {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'AgingReportActions'. Consider to delete it or rename it.

'AgingReportActions' is defined here:


(lint/suspicious/noRedeclare)